style.css

```diff
@@ -0,0 +1,167 @@
+body{
```

Comment:
Please install Prettier in your VSCode for automatic CSS linting! Please let me know if you have issues with the installation! Instructions here: https://swe101.rocketacademy.co/course-logistics/required-hardware-and-software#vscode-formatter-and-settings

Reply:
Done